renderer/views/torrent-list.js

```diff
@@ -217,26 +218,38 @@ function TorrentList (state) {
     }
 
     // Second, render the file as a table row
+    var isPlayable = TorrentPlayer.isPlayable(file)
     var infoHash = torrentSummary.infoHash
     var icon
-    var rowClass = ''
     var handleClick
-    if (TorrentPlayer.isPlayable(file)) {
+    if (isPlayable) {
       icon = 'play_arrow' /* playable? add option to play */
       handleClick = dispatcher('play', infoHash, index)
     } else {
       icon = 'description' /* file icon, opens in OS default app */
-      rowClass = isDone ? '' : 'disabled'
       handleClick = dispatcher('openFile', infoHash, index)
     }
+    var rowClass = 'clickable'
+    if (!isSelected) rowClass = 'disabled' // File deselected, not being torrented
+    if (!isDone && !isPlayable) rowClass = 'disabled' // Can't open yet, can't stream
     return hx`
-      <tr onclick=${handleClick} class='${rowClass}'>
-        <td class='col-icon'>
+      <tr>
+        <td class='col-icon ${rowClass}' onclick=${handleClick}>
```

Comment:
I wonder if we can register `handleClick` once on the `<tr>` tag and just have the onclick handler on `col-select` work? Because dispatcher calls `event.stopPropagation()` we don't need to worry about bubbling.